doc/source/examples/incompressible-flow/2d-taylor-couette-flow-nitsche/2d-taylor-couette-flow-nitsche.rst

```diff
@@ -0,0 +1,358 @@
+A value of 10 is reasonable. Then, we specify the ``number of solids`` geometries that with be represented with Nitsche IB. 
+For each Nitsche IB, a mesh representing the immersed solid has to be specified. Additionally, the solid velocity of the Nitsche IB is specified using the ``solid velocity`` 
+subsection. Finally, the motion of the particle is disabled. This means that even if the immersed particles have a non-zero velocity, they will not physically move in the fluid domain. In this case, this is because our problem has rotation symmetry and we will be seeking steady-state solutions. We note that in this problem, the Nitsche solid grid has the same dimension as the background grid. This is necessary for 2D simulations. Additionally, the Nitsche solid grid is well-refined to ensure that at approximately each fluid cell contains one particle of the immersed body. Finally, we enable the calculation of the torque on the Nitsche IB by setting calculate torsque on solid to ``true``. 
+
+The following figure illustrates the background mesh as well as the particles used to represent the IB on top of it:
+
+.. image:: images/taylor_couette_with_nitsche_mesh.png
+    :alt: The background mesh and the embedded particles
+    :align: center
+    :name: nitsche_particles
+
+Boundary conditions
+~~~~~~~~~~~~~~~~~~~
+
+The ``boundary conditions`` subsection becomes simple since the inner cylinder boundaries were specified in the previous section.
+
+.. code-block:: text
+
+  subsection boundary conditions
+    set number                  = 1
+      subsection bc 0
+          set type              = noslip
+      end
+  end
+
+
+First, the ``number`` of boundary conditions to be applied must be specified. For each boundary condition, the ``id`` of the boundary as well as its ``type`` must be specified. The outer cylinder (``0``) is static and, consequently, a ``noslip`` boundary condition is applied. 
+
+Physical properties
+~~~~~~~~~~~~~~~~~~~
+
+The analytical solution for the Taylor-Couette problem is only valid at low Reynolds number. We thus set the kinematic viscosity to 1.
+
+.. code-block:: text
+
+  subsection physical properties
+    subsection fluid 0
+      set kinematic viscosity            = 1.0
+    end
+  end
+
+
+FEM interpolation
+~~~~~~~~~~~~~~~~~
+
+Lethe supports the use of arbitrary interpolation order. The :math:`\mathcal{L}^2` norm of the error is :math:`\mathcal{O}\left(h^{n+1} \right)` where :math:`h` is a measure of the element size and `n=1` is the interpolation order of the velocity. However, since the torque applied on the inner cylinder depends on the deviatoric stress tensor, which depends on the velocity gradient, it's error will be :math:`\mathcal{O}(n)`. Taking this into account, we use second order polynomials in this example to obtain higher accuracy on the torque. We specify the interpolation order for both pressure and velocity using the ``FEM`` subsection:
```

Comment:
subsection FEM
    set velocity order            = 1
    set pressure order            = 1
end

Reply:
So since using a GLS solver, you can either use a Q1Q1 elements or Q2Q1 elements. If you use Q1Q1 elements, you will reach a second order convergence for velocity, and if using Q2Q1, you will reach a third order (n+1)

Reply:
I added a precision!